std/string.d

```diff
@@ -3734,12 +3734,6 @@ auto leftJustifier(Range)(Range r, size_t width, dchar fillChar = ' ')
             size_t len;
 
           public:
-            this(Range input, size_t width, dchar fillChar)
```

Comment:
> Because it is unnecessary.

It may be, but the change isn't relevant to the focus of this patch.  It'd make for a cleaner changeset to remove the constructor in a separate patch and PR.